inc/plugins/myalerts.php

```diff
@@ -663,18 +663,28 @@ function parse_alert(MybbStuff_MyAlerts_Entity_Alert $alertToParse)
 	$outputAlert = array();
 
 	if ($formatter != null) {
-		$plugins->run_hooks('myalerts_alerts_output_start', $alert);
+		$hook_arguments = [
+			'outputAlert' => &$outputAlert,
+			'formatter' => &$formatter,
+			'alertToParse' => &$alertToParse
+		];
+
+		$plugins->run_hooks('myalerts_alerts_output_start', $hook_arguments);
```

Comment:
Won't this break existing plugins hooking in to MyAlerts by changing the arguments?

Reply:
That was my initial thought too, Euan, but on closer inspection, the existing argument isn't referenced or assigned anywhere else in the function (previously *or* later), so it's currently ineffective anyway.

While we've got your attention: on another matter dealt with by this PR, the removal of the trailing commas in the function calls so as to support PHP < 7.3, there are arguments both ways for this, and it would be useful to know where you stand. On the one hand, as discussed [here](https://community.mybb.com/thread-241473.html), there's the argument that PHP < 7.3 is unsupported and potentially insecure, and we shouldn't tacitly endorse the use of potentially insecure software. On the other, as Omar has pointed out to me, MyBB (1.8) core itself currently officially supports via its requirements PHP >= 5.2, so it's reasonable for plugins to do so too, especially given that this is a minor and otherwise inconsequential change.

Which side do you take?

Reply:
As mentioned the former argument was unset and thus always null, if anything this would be the major change but since it was as useless to begin with any plugin making use of it would be making _wrong_ use of it anyway.